redundant-bmc/src/wait-for-active-passive-target.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
For org purposes, I'd probably move this script under a "scripts" subdir or something but we can always move later if we add more.

Reply:
I'll move it.

Reply:
Done.